systems.csv

```diff
@@ -177,6 +177,11 @@ DE,Potsdam Rad,"Potsdam, DE",nextbike_dc,https://www.nextbike.de/de/potsdam/,htt
 DE,RSVG-Bike,"RSVG, DE",nextbike_rb,https://www.nextbike.de/de/rsvg/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_rb/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_rb%2Fgbfs.json
 DE,RVK,"RVK e-Bike, DE",nextbike_dr,https://www.nextbike.de/de/rvk/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_dr/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_dr%2Fgbfs.json
 DE,SAP Walldorf,"Walldorf, DE",nextbike_ds,https://www.nextbike.de/de/walldorf/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_ds/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_ds%2Fgbfs.json
+DE,Shared Mobility GBFS – CallABike,DE,cab_germany,https://www.callabike.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/CallABike/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FCallABike%2Fgbfs
+DE,Shared Mobility GBFS – CallABikeICE,DE,cab_ice_stationen,https://www.callabike.de/de/ice-stationen,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/CallABikeICE/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FCallABikeICE%2Fgbfs
+DE,Shared Mobility GBFS – StadtRadHamburg,"Hamburg, DE",stadtrad_hamburg,https://stadtrad.hamburg.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/StadtRadHamburg/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FStadtRadHamburg%2Fgbfs
+DE,Shared Mobility GBFS – RegioRadStuttgart,"Stuttgart, DE",regiorad_stuttgart,https://www.regioradstuttgart.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/RegioRadStuttgart/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FRegioRadStuttgart%2Fgbfs
+DE,Shared Mobility GBFS – StadtRADLueneburg,"Lübeburg, DE",stadtrad_lueneburg,https://stadtradlueneburg.de/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/StadtRADLueneburg/gbfs,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapis.deutschebahn.com%2Fdb-api-marketplace%2Fapis%2Fshared-mobility-gbfs%2F2-2%2Fde%2FStadtRADLueneburg%2Fgbfs
```

Comment:
Hello @dersvenhesse,

Thanks again for this contribution to the catalog.

All 5 feeds seem to require authentification. Is it possible for DB to make these feeds publicly available?

By opening your data, you allow developers and researchers to use it to improve shared mobility offerings and increase the discoverability of your services.

Thank you in advance,
Fabien

<img width="451" alt="image" src="https://github.com/MobilityData/gbfs/assets/2423604/04ddb7d6-ddc9-4d80-b328-d2a2481a271f">

Reply:
In my knowledge all listings of https://developers.deutschebahn.com require authentication, so I don‘t think this can be an exception.

Reply:
Thank you for your answer. 

I just need to confirm with the team whether we should list feeds that require authentification in the catalog.

PS: I requested credentials. I look forward to trying the API!

Reply:
@dersvenhesse How do you use the [online validator](https://gbfs-validator.mobilitydata.org/) on these feeds? 

The [DB documentation](https://developers.deutschebahn.com/db-api-marketplace/apis/product/shared-mobility-gbfs/api/79137#/SharedMobilityGBFS_10536/overview) indicates to use the headers `DB-Api-Key` and `DB-Client-Id` but I don't think there is such authentification method in the online validator yet:

<img width="625" alt="image" src="https://github.com/MobilityData/gbfs/assets/2423604/fe8f688b-849d-45da-9a32-0dc9026c69e0">

Thanks!

Reply:
https://github.com/MobilityData/gbfs-validator/pull/128